app/vmui/packages/vmui/src/pages/CustomPanel/QueryConfigurator/QueryConfigurator.tsx

```diff
@@ -176,6 +183,11 @@ const QueryConfigurator: FC<QueryConfiguratorProps> = ({
 
   useEffect(() => {
     onHideQuery && onHideQuery(hideQuery);
+    const hideArgs: Record<string,string> = stateQuery.reduce((acc, current, index) => {
+      acc[`g${index}.hide`] = `${hideQuery.includes(index)}`;
```

Comment:
Don’t write `true` to the query params if it’s the default value. That keeps the URL shorter and cleaner.
Instead of setting every `g{index}.hide`, only add keys where the flag is `false`.